lms/static/sass/shared/_footer.scss

```diff
@@ -358,9 +374,14 @@ $edx-footer-bg-color: rgb(252,252,252);
       display: inline-block;
       margin-bottom: ($edx-footer-spacing/2);
     }
+
+    @media print {
```

Comment:
Ok. I tried adding this in via an include statement; not sure what it looks like at the end of the day. You'll want to test.